src/Microsoft.OData.Edm/Validation/EdmError.cs

```diff
@@ -21,16 +21,33 @@ public class EdmError
         /// <param name="errorCode">An integer code representing the error.</param>
         /// <param name="errorMessage">A human readable message describing the error.</param>
         public EdmError(EdmLocation errorLocation, EdmErrorCode errorCode, string errorMessage)
+         : this(errorLocation, errorCode, errorMessage, EdmErrorLevel.Error)

```

Comment:
>EdmErrorLevel.Error [](start = 56, length = 19)

EdmErrorLevel.None ?

Reply:
Unknown 

---
In reply to: [423160717](https://github.com/OData/odata.net/pull/1758#discussion_r423160717) [](ancestors = 423160717)

Reply:
Resolved.